srcpkgs/zam-plugins/template

```diff
@@ -0,0 +1,103 @@
+		["zamulticomp"]="ZaMultiComp"
+		["zamulticompx2"]="ZaMultiCompX2"
+		["zamautosat"]="ZamAutoSat"
+		["zamcomp"]="ZamComp"
+		["zamcompx2"]="ZamCompX2"
+		["zamdelay"]="ZamDelay"
+		["zamdynamiceq"]="ZamDynamicEQ"
+		["zameq2"]="ZamEQ2"
+		["zamgeq31"]="ZamGEQ31"
+		["zamgate"]="ZamGate"
+		["zamgatex2"]="ZamGateX2"
+		["zamgrains"]="ZamGrains"
+		["zamheadx2"]="ZamHeadX2"
+		["zamphono"]="ZamPhono"
+		["zamtube"]="ZamTube"
+		["zamverb"]="ZamVerb"
+	)
+	declare -A comments=(
+		["zamaximx2"]="Acts as a brickwall limiter for mastering in its default state, but can also be tweaked to raise the average level as a stereo maximizer without ever clipping"
+		["zamulticomp"]="Mono multiband compressor, with 3 adjustable bands."
+		["zamulticompx2"]="Stereo version of ZaMultiComp, with individual threshold controls for each band and real-time visualisation of comp curves."
+		["zamautosat"]="An automatic saturation plugin, has been known to provide smooth levelling to live mic channels."
+		["zamcomp"]="A powerful mono compressor strip"
+		["zamcompx2"]="Stereo version of ZamComp with knee slew control"
+		["zamdelay"]="A simple feedback delay unit with sync-to-host BPM feature and filter."
+		["zamdynamiceq"]="A dynamic equalizer that changes its gain based on detecting a narrow band of frequencies."
+		["zameq2"]="Two band parametric equaliser with high and low shelving circuits."
+		["zamgeq31"]="31 band graphic equaliser, good for eq of live spaces, removing unwanted noise from a track etc."
+		["zamgate"]="Gate plugin for ducking low gain sounds."
+		["zamgatex2"]="Gate plugin for ducking low gain sounds, stereo version."
+		["zamgrains"]="Granular Synthesizer"
+		["zamheadx2"]="HRTF acoustic filtering plugin for directional sound."
+		["zamphono"]="A collection of phono filters for restoring vinyl records, or preparing to cut new ones."
+		["zamtube"]="Wave digital filter physical model of a triode tube amplifier stage, with modelled tone stacks from real guitar amplifiers"
+		["zamverb"]="Reverb"
+	)
+	declare -A generic=(
+		["zamaximx2"]="Maximizer and brickwall limiter"
+		["zamulticomp"]="Mono Multiband Compressor"
+		["zamulticompx2"]="Stereo Multiband Compressor"
+		["zamautosat"]="Automatic Saturation"
+		["zamcomp"]="Mono Compressor"
+		["zamcompx2"]="Stereo Compressor"
+		["zamdelay"]="Delay"
+		["zamdynamiceq"]="Dynamic Equalizer"
+		["zameq2"]="2 Band Parametric Equalizer"
+		["zamgeq31"]="31 Band Graphic Equalizer"
+		["zamgate"]="Mono Gate"
+		["zamgatex2"]="Stereo Gate"
+		["zamgrains"]="Granular Synthesizer"
+		["zamheadx2"]="HRTF Acoustic Filtering"
+		["zamphono"]="Phono Filters"
+		["zamtube"]="Tube Amplifier"
+		["zamverb"]="Reverb"
+	)
+
+	# Generate *.desktop files for each plugin
+	for name in "${_names[@]}"; do
+		gendesk -n \
+			--pkgname "com.zamaudio.${name}" \
+			--name "${name}" \
+			--exec "${exec_names[$name]}" \
+			--pkgdesc "${comments[$name]}" \
+			--genericname "${generic[$name]}"
+	done
```

Comment:
Again, upstream should provide a makefile and/or script, having this logic in template is insanity.

Make upstream handle their side of business, packagers shouldn't have to implement this. Also this should go in `post_build`.

Reply:
OK, I'll remove it from the package and create a PR in upstream repo when I'll have time for that.